src/libs/utils.js

```diff
@@ -197,7 +197,7 @@ export const abandonedPromise = () => {
 
 export const generateRuntimeName = () => `saturn-${uuid()}`
 
-export const generateGalaxyAppName = () => `saturn-k8-${uuid()}`
+export const generateAppName = () => `saturn-k8-${uuid()}`
```

Comment:
I don't think I've ever seen apps actually named like this, is it still used anywhere...?

Reply:
Yes, this is the name that is associated with the app when it is created, and it is used for pause/resume/delete.

Reply:
This name also shows up on the Cloud Environments page under details.

Reply:
`saturn` used to be the name of the UI team. I think we could remove it in favour of something like

```suggestion
export const generateAppName = () => `terra-app-${uuid()}`
```